app/bundles/AssetBundle/Config/config.php

```diff
@@ -88,7 +89,10 @@
                 ],
             ],
             'mautic.asset.reportbundle.subscriber' => [
-                'class' => 'Mautic\AssetBundle\EventListener\ReportSubscriber',
+                'class'     => ReportSubscriber::class,
```

Comment:
Let's use the full namespace here to keep consistency and makes it easier to read the config file. Can you fix UploadSubscriber while here?